szar/clustering.py

```diff
@@ -178,7 +185,7 @@ def ps_tilde(self,mu):
         pklin = pklin[..., np.newaxis]
 
         ans = np.multiply(prefac,pklin.T).T
-        return ans
+        return ans * self.w_redshift_err(mu)
```

Comment:
Should this be squared?
```suggestion
        return ans * self.w_redshift_err(mu)**2
```

Reply:
Done